tests/integration/targets/vmware_rest_appliance/tasks/appliance_health.yml

```diff
@@ -0,0 +1,157 @@
+---
+- name: Get the database health status
```

Comment:
this task is failing, see test results

Reply:
Yes, I know. I think that’s why ignore_errors: true was set in the original test.

Reply:
![Uploading Screenshot 2024-11-06 at 11.37.31.png…]()